### DISCUSSION TITLE
Lora not working

### DISCUSSION THREAD

**Discussion by leandro9000** [vladmandic/sdnext#700] [Q&A] [answered]
I'm having errors when using loras, it used to work normally... now none works anymore! I get the following error in cmd:

output with shape [128, 320] doesn't match the broadcast shape [128, 320, 128, 320]

**Comment by vladmandic** [comment]
i cannot reproduce. are you sure you're using a) normal lora and not something advanced (if so, try using <lyco:filename:strength> to trigger different advanced network) and you're using sd 1.5 model, not 2.0/2.1?

**Comment by storuky** [comment]
Same here. An error occurs with model 2.1 when trying to put LORAs in the Prompt.
Lora is trained on model 2.1 in Kohya (768x768).
#509

**Reply by vladmandic** [reply]
if its 2.x model, then its a duplicate of #509 and its already pretty high on my radar.
@leandro9000 what's the case here?

**Comment by Tillerz** [comment]
I have 3 LORA problems. One thing that worked yesterday:

- if you click a LORA, it gets added to the prompt, if you click it again, it vanishes from prompt again.
NOW if you click it again, it adds it a second time.

After the latest update today I have two problems with LORAs:

- I got a sudden breakage of rendering, where I had to restart the server to fix it. The final output looked like this: https://i.gyazo.com/bad06575e789748e96d22be78ae46fc3.jpg

- I enabled a LORA, did some stuff, disabled it again. But it sticks, it's still being applied despite being removed from the prompt. Is it stuck in the server somehow? Server restart fixed it.

Note for the last two: I also updated the nVidia driver 1 hour ago. Do I have to reinstall the nVidia CUDA package ontop?

**Reply by Anipsy** [reply]
I have a similar issue with LoRas that has began couple hours ago, removing LoRa after use from prompt text window actually does NOT remove it. 
How to reproduce:
Generate a simple prompt - looks normal how it supposed to
Add LoRa  - results change just how they are expected to change with particular LoRa
Remove LoRa - result no longer looks they way it supposed to look without LoRa, it's looks like LoRa still stays loaded, even though it is removed from the prompt.
Add same LoRa again - result looks as if LoRa is applied at twice the weight

Only restarting server fixes it.

I did latest git pull, i did not update/download any other related software on my machine today.

hash: https://github.com/vladmandic/automatic/commit/eb03fce3e4733ef98f8233e5170c12938502674d

**Reply by Anipsy** [reply]
Temporary workaround to flush unwanted LoRas is to change to another model and back.

**Comment by vladmandic** [comment, marked as answer]
i've merged a change that is supposed to fix small memory leak, but it causes this. i'll revert it in ~1h.

**Reply by leandro9000** [reply]
Thanks!! now working fine!

**Comment by Matt541753** [comment]
I have exactly this problem. 1st picture with using a lora is fine (doesn't matter which model and lora), and then from the 2nd picture on it doesn't work anymore and produces only pictures that are poorly rendered, similiar as in the example at this link. https://github.com/vladmandic/sdnext/discussions/700#discussioncomment-5785305

**Reply by vladmandic** [reply]
please report new issue with full details and log, commenting "its the same" as in issue which is 3 years old is not actionable.
